src/components/ButtonMenu/readme.md

```diff
@@ -454,3 +454,254 @@ class ButtonMenuExample extends React.Component {
 }
 <ButtonMenuExample/>
 ```
+
+##### ButtonMenu
+
+```js
+import React from 'react';
+import { ButtonMenu, MenuItem, Card } from 'react-rainbow-components';
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styled from 'styled-components';
+
+const Title = styled.h1.attrs(props => props.theme.rainbow)`
+    font-family: 'Lato Light';
+    font-size: 24px;
+    text-align: left;
+    color: ${props => props.palette.text.main};
+`;
+
+const CardSubtitle = styled.h4.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 12px;
+    color: ${props => props.palette.text.header};
+    display: block;
+`;
+
+const Label = styled.span.attrs(props => props.theme.rainbow)`
+    font-family: Lato Bold;
+    font-size: 24px;
+    line-height: 24px;
+    color: ${props => props.blue? props.palette.brand.main : props.palette.text.main};
```

Comment:
let's use isBrand to indicate a boolean that use the brand color, remember that brand can be any other color